scripts/create_xcframeworks.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+#
+# Shebang to explicitly use bash and not break on macOS boxes with zsh as default shell.
+#
+# After that you usually describe what this script is for and how to use it. In particular,
+# what it expects the current directory to be. (That's more important if it's it "scripts".)
+
+set -eu # common flags to ensure that the shell does not ignore failures
+
+BUILD_PATH=build
+output_dir=$BUILD_PATH/xcf_output
+
+cd ..
+repo_root_path=$(pwd)
+
+mkdir $BUILD_PATH
+mkdir $output_dir
+mkdir $output_dir/archives
+mkdir $output_dir/iphoneos
+mkdir $output_dir/macosx
```

Comment:
If you use `-p` flag, it wouldn't be necessary to create intermediate directories:
```bash
mkdir -p $output_dir/archives
mkdir -p $output_dir/iphoneos
mkdir -p $output_dir/macosx
```